packages/frontend/src/components/RichTextEditor/index.tsx

```diff
@@ -86,11 +89,6 @@ const Editor = ({
       inline: true,
     }),
     StepVariable,
-    Paragraph.configure({
-      HTMLAttributes: {
-        style: 'margin: 0;min-height: 1em;',
-      },
-    }),
   ]
   let content = substituteOldTemplates(initialValue, varInfo) // back-ward compatibility with old values from PowerInput
 
```

Comment:
One nit: if user edits the step again, the `&nbsp;` will show up as a space. 

Not a big deal, but wondering if doign this will fix the issue (not extensively tested, out of scope for this PR)
```
// convert new line character into br elem so tiptap can load the content correctly
content = content.replaceAll('\n', '<br>')
                              .replaceAll('<p style="margin: 0">&nbsp;</p>, '')
```